sound/soc/sdw_utils/sof_sdw_amp_coeff_tables.h

```diff
@@ -0,0 +1,300 @@
+/* SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/*
+ *  sof_sdw_amp_coeff_tables.h - related coefficients for amplifier parameters
```

Comment:
This file name should indicate it is RT specific as well.

Reply:
Agreed. Will rename the file name.